src/views/view.c

```diff
@@ -70,7 +70,11 @@ void dt_view_manager_init(dt_view_manager_t *vm)
 #ifdef HAVE_MAP
                       "map",
 #endif
-                      "slideshow",  NULL };
+                      "slideshow",
```

Comment:
Right, i have seen
option(BUILD_SLIDESHOW "Build the opengl slideshow viewer" ON)
(https://github.com/darktable-org/darktable/blob/5d7890b9cdf01a92a5755749e1537eec3a4d6229/CMakeLists.txt#L17)
and thought that was about slideshow view, and not about that old dtview thingy